README.md

```diff
@@ -44,29 +44,29 @@ This module can be pulled from Maven Central:
 ```
 
 ## Building
-This is a maven project which you can build using
+This is a Maven project which you can build using the following command:
 ```bash
 mvn clean install
 ```
 
 ## Environment
 
-To use these providers you need to have the Python 3.6 with the following packages installed on your environment:
+To use these providers you need to have Python 3.6 with the following packages installed in your environment:
     * numpy
     * scipy
     * jep (this requires JAVA_HOME to be configured)
     * scikit-learn (for the scikit provider)
     
-It should be taken into account that this section only describes the known prerequisites that are common to any model generated in Python.
-Before importing a model you need to ensure the required packages for that model are also installed.
+Note that this section only describes the known prerequisites that are common to any model generated in Python.
+Before importing a model you need to ensure that the required packages for that model are also installed.
 
 ## Running the tests 
 
-To actually run the tests two other configurations may be necessary for Jep to work properly:
+To actually run the tests, two other configurations may be necessary for Jep to work properly:
```

Comment:
the same problem occurs on this bullet list. If you could make this minor fix it would be great :)